scripts/compile-pytorch-ipex.sh

```diff
@@ -173,16 +184,28 @@ if [ "$BUILD_PINNED" = false ]; then
 fi
 
 if [ "$BUILD_FROM_SOURCE" = false ]; then
-  echo "**** Install PyTorch and IPEX from nightly builds. ****"
+  echo "**** Download nightly builds. ****"
   PYTHON_VERSION=$( python -c "import sys; print(f'{sys.version_info.major}.{sys.version_info.minor}')" )
   RUN_ID=$(gh run list -w "Triton wheels" -R intel/intel-xpu-backend-for-triton --json databaseId,conclusion | jq -r '[.[] | select(.conclusion=="success")][0].databaseId')
   TEMP_DIR=$(mktemp -d)
+  WHEEL_PATTERN="wheels-py${PYTHON_VERSION}*"
+  if [ "$UPSTREAM_PYTORCH" = true ]; then
+    WHEEL_PATTERN="wheels-pytorch-py${PYTHON_VERSION}*"
+  fi
   gh run download $RUN_ID \
     --repo intel/intel-xpu-backend-for-triton \
-    --pattern "wheels-py${PYTHON_VERSION}*" \
+    --pattern $WHEEL_PATTERN \
```

Comment:
Just to be on the safe side:
```suggestion
    --pattern "$WHEEL_PATTERN" \
```
We don't want bash to interpolate WHEEL_PATTERN if there is a file or directory with the name that matches .

Reply:
Agree, thank you